Example/Example/AppDelegate.swift

```diff
@@ -248,4 +249,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         self.notify(message)
     }
 
+    func didUpdateToken(_ token: String) {
+        
```

Comment:
Nit, but maybe we'd want `self.notify(token)` here, right?

Reply:
We could. I actually didn't add a call to `startTrackingVerified()` to the example app since there's so much going on in there. But I did want to add the `RadarVerifiedDelegate` as an extra sanity check that things compiled and in case we wanted to add it later